crates/oxc_transformer/src/common/helper_loader.rs

```diff
@@ -182,16 +164,19 @@ impl<'a> HelperLoaderStore<'a> {
             loaded_helpers: RefCell::new(FxHashMap::default()),
         }
     }
+}
 
+// Public methods implemented directly on `TransformCtx`, as they need access to `TransformCtx::module_imports`.
+impl<'a> TransformCtx<'a> {
```

Comment:
Do you think this is a good place to support helper-related methods for `TransformCtx`

Reply:
Just saw your [comment](https://github.com/oxc-project/oxc/pull/6601#issuecomment-2414554481).